fe/fe-core/src/main/java/org/apache/doris/common/FeConstants.java

```diff
@@ -79,4 +79,5 @@ public class FeConstants {
     public static String FS_PREFIX_HDFS = "hdfs";
     public static String FS_PREFIX_FILE = "file";
     public static final String INTERNAL_DB_NAME = "__internal_schema";
+    public static String TEMP_MATERIZLIZE_DVIEW_PREFIX = "doris_internal_tmp_multi_view_";
```

Comment:
I don't think the name `multi view` is suitable, because we just create a one temp view for each job.